bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -125,28 +125,28 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 		return ctrl.Result{}, err
 	}
 
-	// Look up the Machine that owns this KubeConfig if there is one
-	machine, err := util.GetOwnerMachine(ctx, r.Client, config.ObjectMeta)
+	// Look up the owner of this KubeConfig if there is one
+	configOwner, err := util.GetConfigOwner(ctx, r.Client, config.ObjectMeta)
 	if err != nil {
-		log.Error(err, "could not get owner machine")
+		log.Error(err, "could not get owner")
 		return ctrl.Result{}, err
 	}
-	if machine == nil {
-		log.Info("Waiting for Machine Controller to set OwnerRef on the KubeadmConfig")
+	if configOwner == nil {
+		log.Info("Waiting for Owning Controller to set OwnerRef on the KubeadmConfig")
```

Comment:
is "Owning" the actual name of the controller :thinking: ?

Reply:
nope, I changed the string to be more specific. PTAL @neolit123